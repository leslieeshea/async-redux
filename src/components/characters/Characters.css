.Characters {
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;

  li {
    width: 25vw;
    text-align: center;
    list-style-type: none;
  }

  img {
    width: 100%;
    margin: auto;
  }
}
