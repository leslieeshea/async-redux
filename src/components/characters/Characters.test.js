import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('Character component', () => {
  it('renders a character', () => {
    const wrapper = shallow(<Characters characters={[{ id: '1', name: 'aang', img: '/img' }]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
