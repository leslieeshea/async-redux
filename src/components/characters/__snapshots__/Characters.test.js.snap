// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders a character 1`] = `
<ul>
  <li
    key="1"
  >
    <Character
      character={
        Object {
          "id": "1",
          "img": "/img",
          "name": "aang",
        }
      }
    />
  </li>
</ul>
`;
